mitodl/open-discussions
# People widget

#### Pre-Flight checklist


#### What are the relevant tickets?
Most of #1655 

#### What's this PR do?
Adds a new widget for listing people. The widget allows an admin to use a search textbox to list the profiles matching the name and to populate the widget.

#### How should this be manually tested?
Start managing widgets. Add a people widget. Type in a name in the box and you should see at least one suggestion for the profile with that name. Click 'Add' and you should see the suggestion list disappear and the item added below the text field. Click 'create widget' and then 'Done' and you should see the widget with a list of people in it.

You should also be able to edit the widget to add or delete people from it.

#### Screenshots (if appropriate)
![people_widget](https://user-images.githubusercontent.com/863262/52579488-41293300-2df4-11e9-975d-321760911b22.png)

Dialog:
![people_widget_edit](https://user-images.githubusercontent.com/863262/52579502-46867d80-2df4-11e9-984a-e9d269bd9427.png)

![people_widget_selector](https://user-images.githubusercontent.com/863262/52579511-49816e00-2df4-11e9-9708-92abaeb02d8f.png)
